source/features/forked-to.tsx

```diff
@@ -0,0 +1,145 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+import cache from '../libs/cache';
+import {getRepoURL} from '../libs/utils';
+import {isOwnRepo} from '../libs/page-detect';
+
+const currentRepo = getRepoURL();
+
+// Check for cached forks.
+async function checkForks(): Promise<void> {
+	const repo = getSourceRepo();
+	const cached = await getCache(repo);
+	const forks = cached.filter(fork => fork !== currentRepo);
+	appendLink(...forks);
+}
+
+// Check if we are on a forked page.
+function onForkedPage(): void {
+	if (!isOwnRepo()) {
+		return;
+	}
+
+	const forkSourceElement = select<HTMLElement>('.fork-flag:not(.rgh-forked) a');
+	if (forkSourceElement) {
+		const forkedRepo = forkSourceElement.getAttribute('href')!.substring(1);
+		storeCache(forkedRepo, currentRepo);
+	}
+}
+
+// Check for opening the fork dialog.
+function onForkDialogOpened(): void {
+	const forkDialog = select<HTMLElement>('details-dialog[src*="/fork"]')!;
+	const forkFragment = select<HTMLElement>('include-fragment', forkDialog)!;
+	forkFragment.addEventListener('load', () => onFragmentLoaded(forkDialog));
+}
+
+// Event called when fork dialog is opened.
+function onFragmentLoaded(forkDialog: HTMLElement): void {
+	removeLinks();
+
+	const repo = getSourceRepo();
+	const forks = select.all<HTMLElement>('.octicon-repo-forked', forkDialog).map(forkElement => {
+		return forkElement.parentNode!.textContent!.trim();
+	});
+
+	appendLink(...forks);
+	storeCache(repo, ...forks);
+}
+
+// Get the source repo, by checking if we are already on a fork.
+function getSourceRepo(): string {
+	let repo: string;
+
+	const forkSourceElement = select<HTMLElement>('.fork-flag:not(.rgh-forked) a');
+	if (forkSourceElement) {
+		repo = forkSourceElement.getAttribute('href')!.substring(1);
+	} else {
+		repo = currentRepo;
+	}
+
+	return repo;
+}
+
+// Check if the fork still exists.
+async function validateFork(repo: string): Promise<boolean> {
+	const url = new URL(`/${repo}`, location.href);
+	const response = await fetch(String(url),
+		{
+			method: 'HEAD'
+		});
+	return response.ok;
+}
+
+// Get cache and sort it.
+async function getCache(repo: string): Promise<string[]> {
+	const repoKey = key(repo);
+	const cached = await cache.get<string[]>(repoKey) || [];
+	const validForks = cached.filter(validateFork);
+	validForks.sort(undefined);
```

Comment:
Because of `@typescript-eslint/require-array-sort-compare`.

`undefined` is the [explicit default behavior](https://github.com/typescript-eslint/typescript-eslint/issues/247).

Reply:
Sort returns the array, leaving it on one line after filter makes it more clear